rules-tests/TypeDeclaration/Rector/Class_/TypedPropertyFromCreateMockAssignRector/Fixture/nullable_mock_object.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+use Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Source\SomeMockedClass;
+
+class NullableMockObject extends TestCase
+{
+    public $someMock;
+
+    protected function setUp(): void
+    {
+        $this->someMock = $this->createMock(SomeMockedClass::class);
+    }
+
+    protected function tearDown(): void
+    {
+        $this->someMock = null;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+use Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Source\SomeMockedClass;
+
+class NullableMockObject extends TestCase
+{
+    public ?\PHPUnit\Framework\MockObject\MockObject $someMock;
+
+    protected function setUp(): void
+    {
+        $this->someMock = $this->createMock(SomeMockedClass::class);
+    }
+
+    protected function tearDown(): void
+    {
+        $this->someMock = null;
+    }
```

Comment:
Could you also add test fixture with property set in another method except 'setUp()`?

Reply:
do you want to skip if it never assigned in `setUp`, but assigned in another method?

Reply:
see https://github.com/rectorphp/rector-src/pull/6180